util/dvsim/Testplan.py

```diff
@@ -11,43 +11,19 @@
 from pathlib import Path
 
 import hjson
-import logging as log
 import mistletoe
 from tabulate import tabulate
 
 
-class JobTime:
-    # Possible units.
-    units = ["h", "m", "s", "ms", "us", "ns", "ps", "fs"]
-    dividers = [60.0, ] * 3 + [1000.0, ] * 5
-
-    def __init__(self):
-        self.time = 0.0
-        self.unit = "s"
-
-    def __str__(self):
-        if self.time == 0:
-            return ""
-
-        try:
-            index = self.units.index(self.unit)
-        except ValueError as e:
-            log.error("Unsupport JobTime unit: " + self.unit + f"\n{e}")
-            return ""
-
-        # Normalize the time unit.
-        normalized_time = self.time
-        while index > 0 and normalized_time >= self.dividers[index]:
-            normalized_time = normalized_time / self.dividers[index]
-            index = index - 1
-        return f"{normalized_time:.3f}{self.units[index]}"
-
-
 class Result:
     '''The results for a single test'''
 
-    def __init__(self, name, passing=0, total=0, job_runtime=JobTime(),
-                 simulated_time=JobTime()):
+    def __init__(self,
+                 name,
+                 passing=0,
+                 total=0,
+                 job_runtime=None,
+                 simulated_time=None):
```

Comment:
The testplan actually does not need to know the actual type of these parameters. Testplan logic will work fine even if user uses integral, floating and other numeric types. This allows us to move `JobTime` to its own file and not have Testplan even depend on it.